site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -8,376 +8,415 @@ description: 'Instructions on how to attach cloud storage using UI'
 
-        - [`Anonymous access`](https://docs.microsoft.com/en-us/azure/storage/blobs/anonymous-read-access-configure?tabs=portal) -
-          for anonymous access `enable blob public access` in the `configuration` section of your storage account.
-          in this case, you only need the storage account name to gain anonymous access.
-          - `Account name` - storage account name.
+To add access permissions, do the following:
 
-    </br>
+1. Go to [IAM](https://console.aws.amazon.com/iamv2/home#/users) and click **Add users**.
+2. Set **User name** and enable **Access key - programmatic access**.
 
-  - [**Google Cloud**](https://cloud.google.com/docs):
+   ![](/images/aws-s3_tutorial_2.jpg)
 
-    - [`Bucket name`](https://cloud.google.com/storage/docs/creating-buckets) - cloud storage bucket name,
-      you can find the created bucket on the [storage browser page](https://console.cloud.google.com/storage/browser).
+3. Click **Next: Permissions**.
+4. Click **Create group**, enter the group name.
+5. Use search to find and select:
 
-    - `Authorization type`:
+   - For read-only access: **AmazonS3ReadOnlyAccess**.
+   - For full access: **AmazonS3FullAccess**.
 
-      - [`Key file`](#create-a-service-account-and-key-file) - you can drag a key file to the area `attach a file`
-        or click on the area to select the key file through the explorer. If the environment variable
-        `GOOGLE_APPLICATION_CREDENTIALS` is specified for an environment with a deployed CVAT instance, then it will
-        be used if you do not attach the key file
-        ([more about `GOOGLE_APPLICATION_CREDENTIALS`](https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable)).
+   ![](/images/aws-s3_tutorial_3.jpg)
 
-      - [`Anonymous access`](#anonymous-access) - for anonymous access, you need to enable public access to bucket.
+6. (Optional) Add tags for the user and go to the next page.
+7. Save **Access key ID** and **Secret access key**.
 
-    - `Prefix` - used to filter data from the bucket.
+![](/images/aws-s3_tutorial_4.jpg)
 
-    - [`Project ID`](https://cloud.google.com/resource-manager/docs/creating-managing-projects) - you can find
-      the created project on the [cloud resource manager page](https://console.cloud.google.com/cloud-resource-manager),
-      note that the project name does not match the project ID.
+For more information,
+see [Creating an IAM user in your AWS account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html)
 
-    - `Location` - here you can choose a region from the list or add a new one. To get more information click
-      on [`?`](https://cloud.google.com/storage/docs/locations#available-locations).
+#### Anonymous access
 
-    </br>
+On how to grant public access to the
+bucket, see
+[Configuring block public access settings for your S3 buckets](https://docs.aws.amazon.com/AmazonS3/latest/userguide/configuring-block-public-access-bucket.html)
```

Comment:
it looks strange a little bit: how to `grant` but the link to configuring `block`
Need to check maybe there is other documentation about it